File: Bubble_Sort.java
// BUBBLE SORT //////////////////////////////////
        int array[] = {4,2,1,5,3,9,7,6,8};
        boolean sorted = true;

        for (int i=0;i<=(array.length)-1;i++){
            for (int j=0;j<(array.length)-1-i;j++){
                if (array[j+1] < array[j]){
                    int temp = array[j+1];
                    array[j+1] = array[j];
                    array[j] = temp;
                    sorted = false;
                }
            }if (sorted) break;
        }
        for (int items: array) {
            System.out.print(items + " ");
        }

File: sorting/Bubble Sort/bubbleSort.kt
fun main(args: Array<String>) {
    var array = arrayOf(1, 2, 3, 4, 5, 6,12,44,6,887,35,4363,63,3463,3665, 4, 3, 2)

    for (i in 0..array.size - 1) {
        for (j in 0..array.size - 2 - i) {
            if (array[j + 1] < array[j]) {
                var temp = array[j + 1]
                array[j + 1] = array[j]
                array[j] = temp
            }
        }
    }

    for (i in array) {
        print("$i ")
    }
}
